### ISSUE TITLE
Use autoformatter?

### ISSUE THREAD

**Issue by mforets** [JuliaReach/LazySets.jl#3251]
We could use https://www.julia-vscode.org/docs/stable/userguide/formatter/ , potentially with a custom format, then run a script that batch applies to all files. See also the `Format on save` option in VSCode which applies the formatter automatically when saving a file.

**Comment by mforets** [comment]
we may look into YAS style wrt alignment of function signatures > https://domluna.github.io/JuliaFormatter.jl/dev/yas_style/#Differences-from-DefaultStyle

**Comment by mforets** [comment]
>  then run a script that batch applies to all files.

this is actually provided by the package, see 

```julia
# Formats all files in the package directory of a `Module`
julia> format(FooPackage)
```

**Comment by schillic** [comment]
This was fixed in #3320.
